build/makefile_base.mak

```diff
@@ -835,13 +924,17 @@ gst_good: gst_good32 gst_good64
 
 gst_good64: SHELL = $(CONTAINER_SHELL64)
 gst_good64: $(GST_GOOD_CONFIGURE_FILES64)
-	PATH="$(abspath $(TOOLS_DIR64))/bin:$(PATH)" ninja -C "$(GST_GOOD_OBJ64)" install
+	PATH="$(abspath $(TOOLS_DIR64))/bin:$(PATH)" \
+    LD_LIBRARY_PATH="$(abspath $(TOOLS_DIR64))/lib:$(LD_LIBRARY_PATH)" \
```

Comment:
Missing alignment with other lines.

Reply:
You use both spaces and tabs. Spaces are preferred. You can set usage of tab as 4 spaces in your IDE.